src/foam/u2/crunch/wizardflow/CheckPendingAgent.js

```diff
@@ -73,6 +74,10 @@ foam.CLASS({
         this.cancelled = true;
         this.sequence.endSequence();
       }
+      if ( ! shouldReopen && statusPending ) {
+        this.cancelled = true;
+        this.sequence.endSequence();
```

Comment:
**Related to previous comment** - `this.sequence.endSequence()` appears three times in this method now, however I believe it can be placed into a single condition at the end.

Reply:
updated